app/internal/currency.py

```diff
@@ -0,0 +1,14 @@
+import requests
+
+
+def is_valid_currency_api_response(date: str) -> bool:
+    """Checks for valid response from the currency-API request
+    (returns bool)"""
+
+    try:
+        response = requests.get(f'https://api.exchangeratesapi.io/{date}')
```

Comment:
Prefer moving this code to the user end, to prevent high load on the server side :)